mix.exs

```diff
@@ -21,7 +21,7 @@ defmodule Membrane.H264.TODO.Mixfile do
       # docs
       name: "Membrane H264 plugin",
       source_url: @github_url,
-      homepage_url: "https://membraneframework.org",
+      homepage_url: "https://membrane.stream",
```

Comment:
Are we sure we want to change this?

Reply:
Why not? Anyway, `https://membraneframework.org` redirects to `membrane.stream` ;)